iron-form.html

```diff
@@ -316,6 +325,9 @@
 
       /**
        * Resets the form to the default values.
+       *
+       * @param {!Event} event
```

Comment:
Looks like this should be `Event=` instead, as it reads like an optional parameter.